nodeup/pkg/model/ntp.go

```diff
@@ -41,6 +41,9 @@ func (b *NTPBuilder) Build(c *fi.ModelBuilderContext) error {
 	case distros.DistributionCoreOS:
 		klog.Infof("Detected CoreOS; won't install ntp")
 		return nil
+	case distros.DistributionFlatcar:
+		klog.Infof("Detected CoreOS; won't install ntp")
```

Comment:
I am not a reviewer, but you may want to update this also. Looking forward to see a stable Flatcar with newer Docker and kernel versions.
```suggestion
		klog.Infof("Detected Flatcar; won't install ntp")
```

Reply:
I applied the change recommended. Thank you for the interest here @hakman. I hope that this PR will receive the right attention and we can merge it soon.